connexion/decorators/uri_parsing.py

```diff
@@ -96,8 +98,21 @@ def resolve_params(self, params, _in):
         """
         resolved_param = {}
         for k, values in params.items():
+            dict_keys = re.findall(r'\[(\w+)\]', k)
```

Comment:
Can you add some comments about what this does, including links to the specification?

Reply:
done, please check